modular_skyrat/modules/customization/modules/mob/living/silicon/examine.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/silicon/proc/get_silicon_flavortext()
+	. = list()
+	var/flavor_text_link
+	/// The first 1-FLAVOR_PREVIEW_LIMIT characters in the mob's client's silicon_flavor_text preference datum. FLAVOR_PREVIEW_LIMIT is defined in flavor_defines.dm.
+	var/silicon_preview_text = copytext_char((client?.prefs.read_preference(/datum/preference/text/silicon_flavor_text)), 1, FLAVOR_PREVIEW_LIMIT)
+
+	flavor_text_link = span_notice("[silicon_preview_text]... <a href='?src=[REF(src)];lookup_info=open_examine_panel'>Look closer?</a>")
+
+	if (flavor_text_link)
+		. += flavor_text_link
+
+	if(client)
+		var/erp_status_pref = client.prefs.read_preference(/datum/preference/choiced/erp_status)
+		if(erp_status_pref && erp_status_pref != "disabled")
+			. += span_notice("ERP STATUS: [erp_status_pref]")
```

Comment:
Do we really always display the `ERP STATUS:` underneath that string all the time?

Reply:
As far as I can tell examining regular characters, we do.